Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanGraphicsDevice.cpp

```diff
@@ -56,6 +56,7 @@ bool VulkanGraphicsDevice::InitV() {
 
 void VulkanGraphicsDevice::ShutdownV() {
     VULKAN_SAFE_DESTROY_COMMAND_POOL(m_device, m_commandPool, m_allocator);
+    vkDestroyDevice(m_device, NULL);
```

Comment:
`vkCreateDevice` method is failed when before create about 100 devices.
So, we need to destroy device on shoutdown.